lib/api/protocol/release.js

```diff
@@ -0,0 +1,16 @@
+const ProtocolAction = require('./_base-action.js');
+const {isFunction} = require('../../utils');
+/**
```

Comment:
"release()" is too generic, we should call it something like "releaseMouseButton()".